cli/root.go

```diff
@@ -812,7 +812,12 @@ func formatExamples(examples ...example) string {
 	return sb.String()
 }
 
-func (r *RootCmd) checkVersions(i *clibase.Invocation, client *codersdk.Client) error {
+// checkVersions checks to see if there's a version mismatch between the client
+// and server and prints a message nudging the user to upgrade if a mismatch
+// is detected. forceCheck is a test flag and should always be false in production.
+//
+//nolint:revive
+func (r *RootCmd) checkVersions(i *clibase.Invocation, client *codersdk.Client, forceCheck bool) error {
```

Comment:
I think we can solve this without a `forceCheck` bool, will code something here and post

Reply:
Ahh it is because of the `devel` version that always returns a match :thinking:

Reply:
Agreed definitely better